backend/requirements/dev.txt

```diff
@@ -5,7 +5,6 @@
 pytest==3.7.1
 WebTest==2.0.30
 factory-boy==2.11.1
-eth-tester[py-evm]==0.1.0b33
```

Comment:
This is an unrelated removal, but I figure we don't need to include this ETH dep.

Reply:
Nice catch!